rampwf/utils/command_line.py

```diff
@@ -45,10 +50,10 @@ def create_ramp_test_submission_parser():
     parser.add_argument('--pickle', dest='pickle', action='store_true',
                         help='Specify this flag to pickle the submission '
                              'after training.')
-    parser.add_argument('--save-y-preds', dest='save_y_preds',
+    parser.add_argument('--save-output', dest='save_output',
```

Comment:
Since you don't want to break things, maybe you should keep `--save-y-preds` for now but make it point to 
the `save_output` arg, i.e.
```python
parser.add_argument('--save-y-preds', dest='save_output',
...
```
We will remove this argument once we've made sure `--save-y-preds` has be removed from all of the starting kits/docs/wikis.

Reply:
OK, try it.

Reply:
OK, done, it seems to work.